variables.tf

```
@@ -56,6 +56,23 @@ variable "maintenance_window" {
 # Basic RDS
 ###########
 
+variable "instance_availability_zone_list" {
+  description = <<HEREDOC
+List of availability zones to place each aurora instance. Availability zone assignment is by index. The first AZ in the list is assigned to the first instance, 
+second AZ in the list to the second instance, third AZ in the list to the third instance, etc. Variable instance_availability_zone_list_provided must be set to be able to 
+provide list of AZs. Also please remember that the number of AZs specified here should equal to replica_instances + 1.
+HEREDOC
+
+  type    = "list"
+  default = []
```

Based on above comment, we can update this default to 
```suggestion
  default = [""]
```

Should ensure there is an item in the list to lookup, and the `element` method will roll back to the beginning of the list if the index is higher than the list length.